### PR TITLE
Removed CountryGuidePage

### PR DESCRIPTION
**Pull request by webbyfox**
To do (delete all that do not apply):

 - [ ] Change has a jira ticket that has the correct status.
 - [ ] Changelog entry added.
 - [ ] (if there are vulnerable requirements) Upgraded any vulnerable dependencies.
 - [ ] (if updating requirements) Requirements have been compiled.
 - [ ] (if adding env vars) Added any new environment variable to vault.
 - [ ] (if adding feature flags) Cleaned up old flags

### CODE REVIEW

File: activitystream/serializers.py [hunk 1]
```diff
@@ -58,5 +34,5 @@ def to_representation(self, obj):
             return ArticlePageSerializer(obj).data
         elif isinstance(obj, MarketingArticlePage):
             return MarketingArticlePageSerializer(obj).data
-        else:  # CountryGuidePage
-            return CountryGuidePageSerializer(obj).data
+        else:
+            return {}
```
Comment (depsiatwal) [inline activitystream/serializers.py:38]: looks like tests may not be covering this

**Review by depsiatwal — COMMENTED** [review]
just small comment regarding test

File: activitystream/serializers.py (hunk 1 above)
Comment (webbyfox) [reply activitystream/serializers.py]: I have added for safe side but I think its not needed

Comment (webbyfox) [reply activitystream/serializers.py]: Removed @depsiatwal

**Review by depsiatwal — APPROVED** [review]
LGTM

**Pull request merged**